### PR TITLE
Adding rocket, resnet50 and graph_manager updates

### PR DESCRIPTION
**Pull request by naiteeks**
Adding rocket, resnet50 and graph_manager updates

### CODE REVIEW

File: ref-apps/rocket/csharp/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contributing
+
```
Comment (gadamilan) [inline ref-apps/rocket/csharp/CONTRIBUTING.md:2]: We have contributing.md at the root level of the repo. Is there anything extra in this file vs. what is in the root level

File: ref-apps/rocket/csharp/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Copyright (c) Microsoft Corporation.
+
+MIT License
+
```
Comment (gadamilan) [inline ref-apps/rocket/csharp/LICENSE:4]: Is this different form the LICENSE file at the root of the repo?

File: utilities/video-analysis/graph_manager/graph_manager.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import sys
```
Comment (gadamilan) [inline utilities/video-analysis/graph_manager/graph_manager.py:1]: The path for this file doesnt look right.

File: utilities/video-analysis/readme.md [hunk 4]
```diff
@@ -6,11 +6,13 @@ This folder has a set of video analysis related utilities
 
 | Folders              | Description                                       |
 |----------------------|---------------------------------------------------|
+| `graph_manager`      |                                                   |
```
Comment (gadamilan) [inline utilities/video-analysis/readme.md:9]: What is this folder about?

File: ref-apps/rocket/csharp/CONTRIBUTING.md (hunk 1 above)
Comment (naiteeks) [reply ref-apps/rocket/csharp/CONTRIBUTING.md]: no difference. fixed with commit: 62d22e0

File: ref-apps/rocket/csharp/LICENSE (hunk 2 above)
Comment (naiteeks) [reply ref-apps/rocket/csharp/LICENSE]: no difference. fixed with commit: 62d22e0

File: utilities/video-analysis/graph_manager/graph_manager.py (hunk 3 above)
Comment (naiteeks) [reply utilities/video-analysis/graph_manager/graph_manager.py:1]: we created a graph_manager folder to store the graph_manager.py into this folder which is being used by rocket and the yolov samples as well. This way we dont have to duplicate the file in multiple places.

File: utilities/video-analysis/readme.md (hunk 4 above)
Comment (naiteeks) [reply utilities/video-analysis/readme.md:9]: we created a graph_manager folder to store the graph_manager.py into this folder which is being used by rocket and the yolov samples as well. This way we dont have to duplicate the file in multiple places.

**Pull request merged**